buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
I'd lean the other way.  It's cleanup stuff, and as long as we're on the machine, we should try to put the machine in a clean(er) state.